environments/icds-cas/inventory/postgresql.csv

```diff
@@ -38,18 +38,22 @@ pgwarehouse0,100.71.184.23,postgresql,,MUMGCCWCDPRDPW00,,,,
 group,I.var: pgbouncer_max_connections,I.var: pgbouncer_default_pool,I.var: postgresql_max_connections,S.var: postgresql_work_mem,S.var: postgresql_shared_buffers,S.var: postgresql_random_page_cost,,
 postgres,,,,,,,,
 pg_standby,,,,,,,,
-ucr_dbs,3500,450,500,4096kB,12GB,1.1,,
+ucr_dbs,3500,450,500,1872kB,20GB,1.1,,
 shard_dbs,3500,775,800,41943kB,12GB,,,
 main_dbs,3500,775,400,41943kB,12GB,,,
 synclog_dbs,3500,775,800,41943kB,12GB,,,
 plproxy0,3500,775,800,41943kB,8GB,,,
 pgwarehouse0,3500,775,800,655kB,2GB,,,
 ,,,,,,,,
 # group vars for postgres continued,,,,,,,,
-group,I.var: postgresql_checkpoint_segments,F.var: postgresql_checkpoint_completion_target,S.var: postgresql_maintenance_work_mem,S.var: postgresql_wal_buffers,I.var: postgresql_default_statistics_target,S.var: postgresql_effective_cache_size,S.var: postgresql_max_wal_size,S.var: enable_nestloop
-ucr_dbs,128,0.7,2GB,16MB,500,36GB,5GB,off
-shard_dbs,64,0.9,2GB,16MB,100,36GB,,
-main_dbs,32,0.7,2GB,16MB,100,36GB,,
-synclog_dbs,64,0.9,2GB,16MB,100,36GB,,
-plproxy0,64,0.9,2GB,16MB,100,24GB,,
-pgwarehouse0,170,0.9,1GB,16MB,500,6GB,8GB,
+group,F.var: postgresql_checkpoint_completion_target,S.var: postgresql_maintenance_work_mem,S.var: postgresql_wal_buffers,I.var: postgresql_default_statistics_target,S.var: postgresql_effective_cache_size,S.var: postgresql_max_wal_size,S.var: enable_nestloop,
+ucr_dbs,0.9,2GB,16MB,500,60GB,8GB,off,
+shard_dbs,0.9,2GB,16MB,100,36GB,,,
+main_dbs,0.7,2GB,16MB,100,36GB,,,
+synclog_dbs,0.9,2GB,16MB,100,36GB,,,
+plproxy0,0.9,2GB,16MB,100,24GB,,,
+pgwarehouse0,0.9,1GB,16MB,500,6GB,8GB,,
+,,,,,,,,
+# group vars for postgres continued,,,,,,,,
```

Comment:
These changes are hard to follow. Not necessary for this PR, but maybe it would be a positive change to use comments to separate groups of variables. something like:

```
# general memory 
group, S.var: postgresql_maintenance_work_mem, ...

# checkpoints
group, F.var: postgresql_checkpoint_completion_target

# parallel
group,I.var: postgresql_max_worker_processes
```